tests/integration/cloud/providers/test_ec2.py

```diff
@@ -45,7 +45,6 @@ class EC2Test(ShellCase):
     '''
     Integration tests for the EC2 cloud provider in Salt-Cloud
     '''
-    TIMEOUT = 500
```

Comment:
Is there a reason you're not just changing this variable and setting it everywhere in this file?

Reply:
Only because that was not being used consistently within the file.